articles/migrations/guides/unpaginated-api2-requests.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Migrate to Management API v2 Endpoint Paginated Queries
+description: Query requests to specific Management API endpoints will return up to 50 results instead of all available items. You must now specify `page` and `per_page` parameters.
+topics:
+  - pagination
+  - migrations
+contentType:
+  - concept
+  - how-to
+useCase:
+  - paginated-results
+---
+# Migrate to Management API v2 Endpoint Paginated Queries
+
+Beginning on **21 July 2020**, specific Management API v2 endpoints will return up to 50 items instead of all the available items. To retrieve more items, you must include the `page` and `per_page` parameters. The migration period ends on **26 January 2021**.
```

Comment:
@tamigoodall Sorry for so much churn on this... but I don't think this quite accurately describes what's happening. On July 21 we'll start showing tenant logs and a migration toggle for affected customers, no immediate change of behavior.  Maybe something like this along the lines of:

> After January 26, 2021, requests to API v2 will return a maximum of 50 items for tenants in the Public Cloud, to retrieve more items, you must include the `page` and `per_page` parameters. Beginning on 21 July 2020 we'll start displaying tenant logs and a migration toggle to help you prepare for this change.

And I think being explicit that tenants in the private cloud aren't impacted would be useful, since private cloud customers can have public cloud tenants - which would be impacted. Maybe an info box of some sort:

> Only tenants in Auth0's Public Cloud are affected at this time. 

Of course feel free to change the language around as much as you see fit, I just want to make sure we're clear on whats actually changing when.

Reply:
@mmaddex I will make those edits and tweak if necessary.